integrationExamples/gpt/userId_example.html

```diff
@@ -71,10 +71,12 @@
                 t.async = false;
                 t.src = 'http://acdn.adnxs.com/cmp/cmp.bundle.js';
                 var tag = document.getElementsByTagName('head')[0];
-                tag.appendChild(t);
+                tag.appendChild(t);                
             }
         })(window, document);
         // window.__cmp('showConsentTool');
+
+        function getHashedEmail() {}
```

Comment:
Will this file be included in the prebid pr or just in our repo?

Reply:
This is the existing file in their repo that is broken. It's an example page but that method not being defined caused me a bit of confusion